app/src/main/java/org/jellyfin/androidtv/playback/nextup/NextUpActivity.kt

```diff
@@ -50,6 +49,14 @@ class NextUpActivity : FragmentActivity() {
 		}
 	}
 
+	private fun safelyLoadBitmap(url: String): Bitmap? {
+		return try {
+			Glide.with(this@NextUpActivity).asBitmap().load(url).submit().get()
```

Comment:
You might not need the `@NextUpActivity` part here because `this` didn't change.